### PR TITLE
style mobile experience and smooth transitions

### PR DESCRIPTION
**Pull request by slocker3**
- adjust padding for mobile
- adjust font-sizing for mobile
- add script for smooth jump-link scroll
- add script for multiple show/hide items

### CODE REVIEW

**Comment by slocker3** [comment]
@jeremyfelt @philcable #reviewmerge #bump ?

File: style-guide/home.html [hunk 1]
```diff
@@ -293,7 +293,7 @@
 
 								</div>
 						</section>
-						</div><div class="section-wrapper explore-slider">	<section id="builder-section-1479103168782" class="row single ">
+					</div><div id="see" class="section-wrapper explore-slider">	<section id="builder-section-1479103168782" class="row single ">
```
Comment (jeremyfelt) [inline style-guide/home.html:322]: Is this ID used in the JS or CSS at all?

**Review by jeremyfelt — APPROVED** [review]
The changes look fine, but it doesn't look like the style guide has been updated accordingly, so I'm not sure that things actually work. :)

I'd be okay to merge, but it'd be good to have a matching style guide for troubleshooting.

**Comment by jeremyfelt** [comment]
To clarify:
- It looks like there are anchors in production that will work with the scroll/nav JS, so that's probably okay.
- I don't see anything in production or in the style guide for the accordion behavior.

File: style-guide/home.html (hunk 1 above)
Comment (slocker3) [reply style-guide/home.html:322]: Yes. I have added IDs to the other sections that require them

**Comment by slocker3** [comment]
@jeremyfelt @philcable added updates to styleguide

File: src/js/script.js [hunk 2]
```diff
@@ -136,3 +136,40 @@
...
+	        scrollTop: $( $( this ).attr( "href" ) ).offset().top
+	    }, 750 );
+	} );
+} );
+
+( function( $ ) {
+
+	"use strict";
+
+	var $js_accordion = $( ".js-accordion" );
```
Comment (jeremyfelt) [inline src/js/script.js:154]: @slocker3 What section is the accordion in?

**Comment by jeremyfelt** [comment]
:+1:

**Pull request merged**